manage/upgrading/version_specific_migration/upgrade_to_python3.rst

```diff
@@ -306,6 +306,14 @@ With some luck, there are not too many issues with the code left at this point.
 It you are unlucky you have to fix Doctests. These should be changed so that Python 3 is the default. (e.g. text is represented as ``'foo'`` and not ``u'foo'`` and bytes are represented as ``b'bar'`` and not as ``'bar'``). Search for examples of ``Py23DocChecker`` in Plone's packages to find a pattern which allows updated doctests to pass in Python 2.
 
 
+Test your code against buildout.coredev on Jenkins (https://jenkins.plone.org/view/Add-ons/). 
```

Comment:
Can we make that as an nice link, meaning that "buildout.coredev" is the link text ?

Thanks!

Reply:
Fixed in 9f40573.